sky/packages/sky/lib/gestures/drag.dart

```diff
@@ -149,8 +149,8 @@ class PanGestureRecognizer extends _DragGestureRecognizer<sky.Offset> {
 
   sky.Offset get _initialPendingDragDelta => sky.Offset.zero;
   // Notice that we negate dy because scroll offsets go in the opposite direction.
-  sky.Offset _getDragDelta(sky.PointerEvent event) => new sky.Offset(event.dx, -event.dy);
+  sky.Offset _getDragDelta(sky.PointerEvent event) => new sky.Offset(-event.dx, -event.dy);
```

Comment:
I don't think this is correct.  If you try a horizontally scrollable widget, won't it scroll the wrong way now?